src/shared/Entities/modules/signed_certifications.js

```diff
@@ -16,6 +19,20 @@ export function createSignedCertification(
         ...payload,
       },
     },
-    { Label },
+    { label },
   );
 }
+
+export function getSignedCertification(moveId, label = getSignedCertificationsLabel) {
+  return swaggerRequest(getClient, 'certification.indexSignedCertification', { moveId }, { label });
+}
+
+export function selectPaymentRequestCertificationForMove(state, moveId) {
+  const signedCertifications = filter(state.entities.signedCertifications, cert => {
+    return cert.certification_type === 'PPM_PAYMENT' && cert.move_id === moveId;
+  });
+  if (!Array.isArray(signedCertifications) || !signedCertifications.length) {
```

Comment:
```suggestion
  if (!signedCertifications.length) {
```
Filter will return an array no matter what even if signedCertifications is undefined

Reply:
👍